main/webapp/modules/core/langs/translation-en.json

```
@@ -371,6 +371,9 @@
     "core-dialogs/name": "Name",
     "core-dialogs/action": "Action",
     "core-dialogs/should-return-number": "your expression should return a number",
+    "core-dialogs/remove-duplicate-rows": "Remove duplicate rows",
+    "core-dialogs/remove-duplicate-rows-col-select": "Select columns to identify duplicates",
+    "core-buttons/no-criteria-selected": "Row duplicate identification criteria not set. No column(s) selected.",
```

Maybe we can simplify this message a bit, I don't think it needs to be two sentences. How about:
```suggestion
    "core-buttons/no-criteria-selected": "At least one column needs to be selected to identify duplicates.",
```